### ISSUE TITLE
Rack crashes when connecting a clock to Sequence in Mac

### ISSUE THREAD

**Issue by BeZaBel** [hdavid/VCVRack-Holon.ist#2]
When I connect any clock source to the clock input of Sequence Rack crashes immediately. It doesn't happen when connecting to any other input or output in Sequence, and only happens with the Sequence module.

**Comment by hdavid** [comment]
Hi, i will try to reproduce when i get back from holidays.
could you provide me with some details ? 
- which version of the code did you use ? (i suppose you compiled from source, as sequence has not been released yet)
- which module/signal you used as clock source ?

**Comment by BeZaBel** [comment]
Thank you so much! Yes, I compiled from source. And the crash happens as soon as I connect any clock signal to the clock input. I tested it with many of the clocks available: AS' BPM Clock, Bidoo's Tocante clock, Impromptu's Clocked, even various square LFOs like the BPM LFO, the Fundamental LFO, Tri LFO, etc. All crashed when connecting the signal. If the clock is not active it will not crash until I hit run. The crash happens inmediately.
